src/components/CollectionFilters/LineFilter.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as FiltersActions from '../../actions/filters';
+import * as FilterSetsActions from '../../actions/filterSets';
+
+class LineFilter extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  filterIsApplied() {
+    const filters = this.props.filters;
+    for (let i = 0; i < filters.length; i++) {
+      if (filters[i].slug === this.props.filter.slug) {
+        return i;
+      }
+    }
+    return -1;
```

Comment:
Not a huge deal, but let's aim for consistency and add line breaks after blocks. This is a good guide. https://github.com/airbnb/javascript#whitespace--after-blocks

Reply:
👍  I'll go through and make style changes where needed.